packages/account/src/providers/transaction-summary/operations.ts

```diff
@@ -231,6 +240,77 @@ export function getWithdrawFromFuelOperations({
   return withdrawFromFuelOperations;
 }
 
+/** @hidden */
+function getContractCalls(
+  contractInput: InputContract,
+  abiMap: AbiMap | undefined,
+  receipt: TransactionResultCallReceipt,
+  rawPayload: string,
+  maxInputs: BN
+): FunctionCall[] {
+  const abi = abiMap?.[contractInput.contractID];
+  if (!abi) {
+    return [];
+  }
+
+  return [
+    getFunctionCall({
+      abi,
+      receipt,
+      rawPayload,
+      maxInputs,
+    }),
+  ];
+}
+
+/** @hidden */
+function getAssetsSent(receipt: TransactionResultCallReceipt): Asset[] | undefined {
+  return receipt.amount?.isZero()
+    ? undefined
+    : [
+        {
+          amount: receipt.amount,
+          assetId: receipt.assetId,
+        },
+      ];
+}
+
+/** @hidden */
+function processReceipt(
```

Comment:
We can rename this helper to be more specific, like `processCallReceipt`

Reply:
18b6669